backend/src/bundles/resumes/enums/graph-const-relations.enum.ts

```diff
@@ -0,0 +1,4 @@
+const resumeGraphFetchRelations =
```

Comment:
It's not an `enum`. But this file in `enums` folder

Reply:
I change to constant.